src/libs/Violations/ViolationsUtils.ts

```diff
@@ -327,6 +327,8 @@ const ViolationsUtils = {
         const categoryName = updatedTransaction.category;
         const categoryMaxAmountNoReceipt = policyCategories[categoryName ?? '']?.maxAmountNoReceipt;
         const maxAmountNoReceipt = policy.maxExpenseAmountNoReceipt;
+        // We need to negate the amount because the amount is stored with opposite sign
+        const negatedAmount = -amount;
```

Comment:
Why this instead of just replacing in the conditions below `Math.abs(amount)` with `-amount`?

Reply:
For semantic meaning

Reply:
but then you don't need both `Math.abs(amount) > maxAmountNoReceipt` and `negatedAmount > 0`. You can just write `negatedAmount > maxAmountNoReceipt`.